setup-files/setup-visual.el

```diff
@@ -218,7 +218,7 @@ the smart-mode-line theme."
 ;; ;; `after-make-frame-functions' hook is not run in no-window mode
 ;; (add-hook 'after-make-frame-functions (lambda (&rest frame)
 ;;                                         (funcall default-theme-fn)))
-(add-hook 'window-setup-hook (lambda () (funcall default-theme-fn)))
+(add-hook 'window-setup-hook #'(lambda () (funcall default-theme-fn)))
```

Comment:
We should not sharp-quote lambdas. See https://endlessparentheses.com/get-in-the-habit-of-using-sharp-quote.html for reference. It's an old but interesting post on this topic.